cql3/statements/create_keyspace_statement.cc

```diff
@@ -92,11 +92,12 @@ future<std::tuple<::shared_ptr<cql_transport::event::schema_change>, std::vector
     using namespace cql_transport;
     const auto& tm = *qp.proxy().get_token_metadata_ptr();
     const auto& feat = qp.proxy().features();
+    const auto& cfg = qp.db().get_config();
     std::vector<mutation> m;
```

Comment:
> so that even when set to `false`, tablets would
> still be enabled as a feature, so that a following

This certainly shouldn't be backported, it's a large change.